MACOS.md

```diff
@@ -198,14 +184,67 @@ brew upgrade openssl     || brew install openssl
 ```
 
 
-## Sublime Text 3 - Your text editor
+## Visual Studio Code
+
+### Installation
+
+Let's install [Visual Studio Code](https://code.visualstudio.com) text editor.
+
+Copy (`CMD` + `C`) the command below then paste it in your terminal (`CMD` + `V`):
+
+```bash
+brew install --cask visual-studio-code
+```
+
+Then launch VS Code:
+
+```bash
+'/Applications/Visual Studio Code.app'
+```
+
+Now, add VS Code to your Dock by right-clicking on the icon to bring up the context menu and choosing Options, Keep in Dock.
+
+### Launching from the terminal
+
+In VS Code, open the Command Palette (`CMD` + `SHIFT` + `P`), type `shell command` and click on "Shell Command: Install 'code' command in PATH" :point_down:
+
+![](images/mac_vscode_command.png)
+
+Now quit the VS Code, quit the Terminal (`CMD` + `Q`) and restart it.
+
+Try typing `code` :point_right: if VS Code opens in new window, you can proceed to the next point!
+
+
+## VS Code Extensions
+
+### Installation
+
+Let's install some useful extensions to VS Code. Copy the command below then paste it in your terminal:
+
+```bash
+code --install-extension ms-vscode.sublime-keybindings MS-vsliveshare.vsliveshare misogi.ruby-rubocop hridoy.rails-snippets dbaeumer.vscode-eslint
```

Comment:
Multiple install did not work on my machine, I guess it's safer to give instructions to install them one by one❔ 
<img width="1502" alt="Screen Shot 2021-04-04 at 19 14 23" src="https://user-images.githubusercontent.com/8090140/113516450-9ac3bd00-957a-11eb-929a-d0ced4310a02.png">

Reply:
Good to know. Thanks. I've rolled back to one-by-one installation instructions :heavy_check_mark: